pkg/remote/terraform/provider.go

```diff
@@ -24,13 +25,16 @@ import (
 	tf "github.com/cloudskiff/driftctl/pkg/terraform"
 )
 
+const LockFileName = ".terraform.lock.hcl"
```

Comment:
I don't think we need this exported const since this is only used at 1 place below.